actionview/lib/action_view/railtie.rb

```diff
@@ -33,6 +33,8 @@ class Railtie < Rails::Railtie # :nodoc:
         if app.config.action_view.cache_template_loading.nil?
           ActionView::Resolver.caching = app.config.cache_classes
         end
+
+        PartialRenderer.collection_cache = Rails.cache
```

Comment:
You've declared `collection_cache` to be a `mattr_reader`. There doesn't seem to be a corresponding writer.